R/imports.R

```diff
@@ -24,60 +24,30 @@ NULL
 #' @export
 dplyr::mutate
 
-#' @importFrom dplyr mutate_
-#' @name mutate_
-#' @rdname reexports
-#' @export
-dplyr::mutate_
```

Comment:
What's your reasoning for removing these exports?

Reply:
I just thought that importing the SE verbs would be unnecessary since we were switching to NSE-only. Should I put them back in?

Reply:
These are "re-exports" meaning that it allows you to do (without attaching dplyr)

```r
library(plotly)
plot_ly(txhousing) %>% 
  group_by_("city") %>% 
  add_lines(x = ~date, y = ~median)
```

I doubt a lot of people are using this, but since this could break user code, we should keep the re-exports